Makefile

```diff
@@ -119,16 +127,20 @@ clean:
 	@find ./ -name '*~' -exec rm -f {} \;
 
 .PHONY: version
-## dump package name into VERSION env variable and show
+## dump version into VERSION env variable and show
```

Comment:
This command is not doing what you are saying here. It only prints in the terminal the content of the variable VERSION already defined [at the beginning of this file](https://github.com/quintoandar/butterfree/pull/286/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R4)

Reply:
I can change the description, but I didn't change the behavior of the command so I kept the description :thinking:

Reply:
you did change, you removed the command that was dumping the version from setup.py into the env var, now being executed at the start of the makefile, so now this command doesn't dump, it just prints... changing the description works :)